index.html

```diff
@@ -34,15 +34,7 @@ <h1>I’m Tshepo. <br> Glad to see <br> you!</h1>
   
 </div>
 
-
-
-
-
-
-   
-
-
-    <div class="Container2">
+   <div class="Container2">
       <img class="Snapshoot2" src="image/SnapshootD3.png" alt="">
```

Comment:
- [ ] Your images are missing an `alt attribute`. :thinking: Currently you left them empty, following the best practices, all images are meant to have an alt tag. **The purpose of alt text is to improve accessibility by describing what an image is showing to visitors who do not have the ability to see them. However, it also helps search engine crawlers and so improves SEO.** :smile: :+1: